extensions/wikia/UserRenameTool/process/ProcessGlobal.class.php

```diff
@@ -0,0 +1,597 @@
+<?php
+
+namespace UserRenameTool\Process;
+
+use UserRenameTool\Tasks\MultiWikiRename;
+use Wikia\Tasks\Queues\PriorityQueue;
+
+class ProcessGlobal extends ProcessBase {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "ProcessGlobal" has 24 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)